persistence/upgrades/sql-5to6.md

```diff
@@ -0,0 +1,19 @@
+---
+title: SQL Persistence Upgrade Version 5 to 6
+summary: Instructions on how to upgrade to SQL Persistence version 6
+reviewed: 2020-06-25
+component: SqlPersistence
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+---
+
+## Move to .NET 4.6.1
+
+The minimum .NET Framework version for version 5 is [.NET 4.6.1](https://dotnet.microsoft.com/download/dotnet-framework/net461).
+
+**All projects must be updated to .NET Framework 4.6.1 before upgrading to version 5.**
+
+It is recommended to update to .NET Framework 4.6.1 and perform a full migration to production **before** updating to version 5. This will help isolate any issues that may occur.
```

Comment:
```suggestion
It is recommended to update to .NET Framework 4.6.1 and perform a full migration to production **before** updating to version 6. This will help isolate any issues that may occur.
```